main.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/weaveworks/metal-janitor-action/action"
+)
+
+func main() {
+	action.Log("running metal janitor action")
+
+	input, err := action.NewInput()
+	if err != nil {
+		action.LogErrorAndExit("failed parsing action input: %s", err.Error())
```

Comment:
All of these can be simplified to just `err` rather than `err.Error()` as the formatter will call `Error` on err objects

https://golang.org/pkg/fmt/#hdr-Printing

> 4. If an operand implements the error interface, the Error method will be invoked to convert the object to a string, which will then be formatted as required by the verb (if any).